lib/routes/podcast/sleep_selector.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:anytime/bloc/podcast/audio_bloc.dart';
+import 'package:anytime/l10n/L.dart';
+import 'package:anytime/state/sleep_state.dart';
+import 'package:anytime/ui/widgets/sleep_selector.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:provider/provider.dart';
+
+class BreezSleepSelector extends SleepSelector {
```

Comment:
Why do we need to override this class?

Reply:
Good question, there are two reasons:

- The icon on AnytimePlayer is a standard icon of the material, a crescent moon, and it is filled or not depending on the activated status of the sleep timer in the other hand here on Breez it is an SVG icon without any changes on its state.
- The second is the `dialogItemTileRadioFillColor`

If we manage to move the dialogItemTileRadioFillColor to the theme data, I can think in a way to pass the icon to the original widget so we can remove this overridden widget.

Reply:
Yes, I think what you suggest is a good approach.

Reply:
BTW even if overriding the theme doesn't work it is more compact and simple  to pass both icon and color via constructor instead of inheritance.

Reply:
That is true, I'll change that 👍

Reply:
✅   Now the theme is properly set and the `SleepSelector` receives some properties to be customizable 🥳